src/reference-manual/pathfinder.qmd

```diff
@@ -25,3 +25,29 @@ evaluations, with greater reductions for more challenging posteriors.
 While the evaluations in @zhang_pathfinder:2022 found that
 single-path and multi-path Pathfinder outperform ADVI for most of the models in the PosteriorDB evaluation set,
 we recognize the need for further experiments on a wider range of models.
+
+## Diagnosing Pathfinder
+
+Pathfinder diagnoses the accuracy of the approximation by computing the density ratio of the true posterior and 
+the approximation and using Pareto-$\hat{k}$ diagnostic (Vehtari et al., 2024) to assess whether these ratios can
+be used to improve the approximation via resmapling. /, the
+normalization for the posterior can be  estimated reliably (Section 3, Vehtari et al., 2024), which is the
+first requirement for reliable resampling.  If estimated Pareto-$\hat{k}$ for the ratios is smaller than 0.7,
+there is still need to further diagnose importance sampling estimates by taking into account also the expetant
```

Comment:
I don't know the word "expetant"---looks like it's Latin.  Can we just write this in English?